src/matrix3.js

```diff
@@ -73,40 +61,54 @@ export default class Matrix3 extends MathArray {
   }
 
   /* eslint-disable max-params */
-  setRowMajor(m00 = 1, m10 = 0, m20 = 0, m01 = 0, m11 = 1, m21 = 0, m02 = 0, m12 = 0, m22 = 1) {
-    return this.set(m00, m01, m02, m10, m11, m12, m20, m21, m22);
+  setRowMajor(m00 = 1, m01 = 0, m02 = 0, m10 = 0, m11 = 1, m12 = 0, m20 = 0, m21 = 0, m22 = 1) {
```

Comment:
Names of the arguments should match their position in the matrix.

Reply:
I just compared it with the implementation in `matrix4`. It seems the naming in the `setColumnMajor` is wrong. In `setRowMajor`, the matrix passed in should follow the row major indices. in `setColumnMajor`, it follows the column major indices.